internal/planning/issue-3885-implementation-guide.html

```
@@ -0,0 +1,665 @@
+
+      <!-- Layer 2: streamingUtils -->
+      <rect x="30" y="120" width="820" height="55" rx="8" fill="var(--surface2)" stroke="var(--orange)"/>
+      <text x="50" y="142" fill="var(--orange)" font-size="11" font-weight="700">② streamingUtils.ts — cancelUpstream()</text>
+      <text x="50" y="162" fill="var(--text-dim)" font-size="10">Same mechanism: readableStream 'close' → cancelUpstream() → fires onConsumerAbort handlers</text>
+
+      <line x1="440" y1="175" x2="440" y2="195" stroke="var(--text-dim)" stroke-width="1.5" marker-end="url(#arrD)"/>
+
+      <!-- Layer 3: proRSC -->
+      <rect x="30" y="200" width="820" height="70" rx="8" fill="var(--surface2)" stroke="var(--cyan)"/>
+      <text x="50" y="222" fill="var(--cyan)" font-size="11" font-weight="700">③ proRSC.ts — onConsumerAbort handler</text>
+      <text x="50" y="242" fill="var(--text-dim)" font-size="10">onConsumerAbort(() => { rscRequestTracker.clear(); postSSRHookTracker.notifySSREnd(); })</text>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)